### DISCUSSION TITLE
Onboarding not triggered on new repositories

### DISCUSSION THREAD

**Discussion by DanySK** [renovatebot/renovate#12163] [Request Help]
By reading the docs, my understanding is that if I create new repositories then Renovate should automatically open an onboarding PR. It is not the case for me: repositories that I had *before* enabling Renovate got their onboarding just right, those instead which have been created later have not.
For instance: https://github.com/DanySK/action-checkout was created more than a month ago, it shows up on the renovate dashboard, but it has never been opened an onboarding PR: https://app.renovatebot.com/dashboard#github/DanySK/action-checkout.

I know how to work this around (adding `renovate.json` manually works), but I'd like to inspect what it is going to happen rather than enabling and receiving actual update PRs upfront. Any help?

**Comment by rarkins** [comment]
What's going on is that Renovate decided a repo was "disabled" (e.g. no package files) and then after that it gets rarely retried (we periodically enqueue up every single repo including forks and archived ones to check them again). Just now I manually reset every one of your repos which was disabled so you should see them all run within the next 2-3 hours

**Reply by DanySK** [reply]
It works, confirmed. So do I understand correctly that the reason is that Renovate thought that there was no file it could run updates on? That would make sense for that repo, as the ability to update `action.yml` files is pretty recent.

**Reply by rarkins** [reply]
It means:
- It ran - probably once - and found zero dependencies - so it didn't attempt to onboard
- The repo got marked as "disabled" in the Renovate backend DB
- At some point some dependencies got added in
- No "background" job got queued up since then, so it remains disabled

**Reply by DanySK** [reply]
Clear, thanks
